.changeset/new-doors-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/combobox": major
+---
+
+Adding read only state to combobox
```

Comment:
I'm not sure this is a major change, but double check with Patrick & Cassondra. A major change would result in a change in version numbers, and breaking changes. Since most of this is additive and, with the small amount of time I've spent with this branch, doesn't change existing behaviors, are they still considered "breaking changes?" 

(asking for a friend 😆)

Reply:
You're right, it is not a breaking change. I thought since it is a feature that devs should be aware of I made it `major` .